dart/welcome_email/main.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:http/http.dart' as http;
+
+Future<void> start(final req, final res) async {
+  final domain = req.env['MAILGUN_DOMAIN'];
+  final mailgun = MailgunMailer(
+    apiKey: req.env['MAILGUN_API_KEY']!,
+    domain: domain ?? '',
```

Comment:
Can we default to EU? or US? Or throw error if not provided?

Reply:
I'm not sure what you mean by this

Reply:
`domain: domain ?? '',` here we default to empty string. Is that on purpose? Is there some default value we could use? Maybe I miss-understood the code. If it's not relevant, feel free to resolve without any actions.

Reply:
I think this is the domain used to send the email, I'll add a validator for that too.